src/apps/relay/netengine.c

```diff
@@ -748,10 +752,10 @@ static int handle_relay_message(relay_server_handle rs, struct message_to_relay
       ioa_socket_handle s = sm->m.sm.s;
 
       if (!s) {
-        TURN_LOG_FUNC(TURN_LOG_LEVEL_ERROR, "%s: mobile socket EMPTY\n", __FUNCTION__);
+        TURN_LOG_CATEGORY("relay", TURN_LOG_LEVEL_ERROR, "%s: mobile socket EMPTY\n", __FUNCTION__);
       } else if (s->read_event || s->bev) {
-        TURN_LOG_FUNC(TURN_LOG_LEVEL_ERROR, "%s: mobile socket wrongly preset: 0x%lx : 0x%lx\n", __FUNCTION__,
-                      (long)s->read_event, (long)s->bev);
+        TURN_LOG_CATEGORY("relay", TURN_LOG_LEVEL_ERROR, "%s: mobile socket wrongly preset: 0x%lx : 0x%lx\n",
+                          __FUNCTION__, (long)s->read_event, (long)s->bev);
```

Comment:
__FUNCTION__ seems like something that should be populated into the log function call automatically using a macro.

Can the macro be adjusted to always populate the function and line number?